src/commands/shared.ts

```diff
@@ -202,12 +211,16 @@ export const requestToSsh = (request: Request<PluginRequest>): SshRequest => {
       type: "aws",
     };
   } else if (request.permission.spec.type === "gcloud") {
```

Comment:
This appears to violate separation of concerns.

Would make more sense I think to add SSH-specific callbacks in the two CSP plugins, and look those up from here.